spec/unit/block_score/collection_spec.rb

```diff
@@ -26,20 +26,33 @@
     end
 
     it { is_expected.to be_empty }
-    its(:class) { should be described_class }
+    it { is_expected.to be_an_instance_of described_class }
   end
 
-  describe '.create' do
+  describe '#create' do
     before { question_sets.create }
 
     it { is_expected.not_to be_empty }
     its(:count) { should be 1 }
-    its(:class) { should be described_class }
+    it { is_expected.to be_an_instance_of described_class }
   end
 
-  describe '.retrieve' do
+  describe '#retrieve' do
     let(:question_set_id) { create(:question_set, person_id: person.id).id }
-
-    it { expect(question_sets.retrieve(question_set_id).class).to be BlockScore::Collection::Member }
+    let(:question_set_id2) { create(:question_set, person_id: person.id).id }
+
+    it 'is the proper class' do
+      expect(question_sets.retrieve(question_set_id))
+        .to be_an_instance_of BlockScore::Collection::Member
+      expect(question_sets.retrieve(question_set_id))
+        .to be_an_instance_of BlockScore::Collection::Member
+      expect(question_sets.count).to equal(1)
+
+      expect(question_sets.retrieve(question_set_id2))
+        .to be_an_instance_of BlockScore::Collection::Member
+      expect(question_sets.retrieve(question_set_id2).id)
+        .to eql question_set_id2.dup
+      expect(question_sets.count).to equal(2)
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/71/files#r52862894